src/Flexbox.jsx

```diff
@@ -1,39 +1,38 @@
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import styled from 'styled-components';
 
-const { string, oneOf, node, number, bool, object } = PropTypes;
```

Comment:
I personally prefer prefixing with `PropTypes.` because some of those names are too generic and I like to make it obvious where do they come from: `string`, `number`, `object`, etc.